platforms/cpus/rs_virgo.repl

```diff
@@ -23,11 +23,7 @@ flash: Memory.MappedMemory @ sysbus 0xB0000000
 // =======================
 button1: Miscellaneous.Button @ sysbus
```

Comment:
Delete this button definition, as all buttons and leds are expected to be in the rs_virgo-eval.repl

Reply:
this button defination is already a part of master_rs for testing scu . So , should I remove it ?

Reply:
Yes, please